Pipfile

```diff
@@ -7,10 +7,12 @@ verify_ssl = true
 flake8 = "*"
 pytest = "*"
 pytest-django = "*"
+django-filter = "*"
```

Comment:
not seeing a mention of this change in the PR comment - why is this needed?

Reply:
I 

> not seeing a mention of this change in the PR comment - why is this needed?

@ifireball 
I mentioned it in the last commit here. 
I needed it because I used filter view for the questions view in order to enable filtering questions in the future  (I split this PR so the form that enables filtering questions is in PR  #83  ).